### PR TITLE
Address review feedback from litherum

### PR DESCRIPTION
**Pull request by FrankYFTang**
@litherum close feedback from litherum on https://github.com/tc39/proposal-intl-displaynames/pull/30

### CODE REVIEW

**Review by leobalter — APPROVED** [review]

**Comment by sffc** [comment]
@litherum can you take a look?

**Review by leobalter — APPROVED** [review]

**Comment by leobalter** [comment]
LGTM. The git conflict can be easily fixed with a new build of the index.html file.

**Comment by leobalter** [comment]
somehow this is my second review. @FrankYFTang I believe it's ok to merge this for now if you're comfortable with the changes. There isn't anything new in the normative context here.

**Pull request merged**
